### PR TITLE
fix: box sizing mixin

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Adding box-sizing mixin and adding it to component that do not have that rule.

**Solving issue**  
Fixes:  Issue with Footer

**How to test**  
1. Go to Netlify link 
2. Check mainly footer component, but also others if they are displayed correctly

### CODE REVIEW

File: tegel/src/components/side-menu/side-menu.stories.ts [hunk 1]
```diff
@@ -114,11 +114,11 @@ const Template = ({ showIcons, collapsed }) => {
       <li class="sdds-sidebar-nav__item sdds-sidebar-nav__extended">
       <button class="sdds-sidebar-nav__item-link">
       <div>
-        <sdds-icon class="sdds-sidebar-nav__icon" name="truck" size="20px></sdds-icon> 
+        <sdds-icon class="sdds-sidebar-nav__icon" name="truck" size="20px"></sdds-icon>
```
Comment (mJarsater) [inline tegel/src/components/side-menu/side-menu.stories.ts:117]: Nice catches!

File: tegel/src/components/divider/divider.scss [hunk 2]
```diff
@@ -7,17 +7,20 @@ $border-types: top, right, bottom, left;
   .sdds-divider-#{$type} {
     background-color: var(--sdds-divider-#{$type});
     height: 1px;
+    box-sizing: border-box;
```
Comment (mJarsater) [inline tegel/src/components/divider/divider.scss:10]: Should we use the mixin here also?

**Comment by mJarsater** [comment]
Something is off with the Native button with Icon. Not sure if it was this PR that introduces it.

File: tegel/src/components/divider/divider.scss (hunk 2 above)
Comment (theJohnnyMe) [reply tegel/src/components/divider/divider.scss:10]: There are more components that have it coded like that, I left it on purpose. I guess we could say that this mixin is fixed for now, but later on, we should have some kind of reset mixin to include it in all components, same as you were mentioning with padding and margin things.

**Comment by mJarsater** [comment]
> Something is off with the Native button with Icon. Not sure if it was this PR that introduces it.

Seems like the sdds-icon has the wrong margin-left, should be 28px for lg, 20px for md and 24 for sm. Should we fix it in this or in another PR?

File: tegel/src/components/divider/divider.scss (hunk 2 above)
Comment (mJarsater) [reply tegel/src/components/divider/divider.scss:10]: We could add a:

`div[class^='sdds-'], div[class*=' sdds-']{
  box-sizing: border-box;
}`

In the global.scss I guess?

**Comment by theJohnnyMe** [comment]
@mJarsater - I will try to fix it in this one, both the issues. Thank you for good 👀

**Review by alexandereneroth — APPROVED** [review]
Tons of changes, didn't test all components, but couldn't find anything broken.
LGTM 👍

**Pull request merged**